RecoMuon/MuonIdentification/plugins/ME0SegmentMatcher.cc

```diff
@@ -164,6 +164,8 @@ void ME0SegmentMatcher::produce(edm::Event& ev, const edm::EventSetup& setup) {
 
       //Remove later
       if (std::abs(thisTrack->eta()) < 1.8) continue;
+      // low pt tracks cannot be propagated to me0
+      if (thisTrack->pt() < 1.0) continue;
```

Comment:
Hi Slava,

I didnt really check what the min was and assumed a lower pt wouldnt be used.
I think using pt would be the easiest and fastest way.
Do you have a better idea?